backend/models/leaderboard.model.js

```diff
@@ -0,0 +1,27 @@
+import mongoose from "mongoose";
```

Comment:
Why do name it leaderboard.model.js. My system does not like the two dots..? Maybe just use leaderboardModel....camelcase

Reply:
I will do this without two dots then.